WindowsServerDocs/identity/ad-cs/configure-trusted-roots-disallowed-certificates.md

```diff
@@ -0,0 +1,401 @@
+
+1. In the Certificate Export Wizard, select **Next**.
+
+1. On the **Export File Format** page, select **Microsoft Serialized Certificate Store (.SST)**, and then select **Next**.
+
+1. On the **File to Export** page, enter a file path and an appropriate name for the file, such as **C:\\AllowedCerts.sst**, and then select **Next**. Select **Finish**. When you're notified that the export was successful, select **OK**.
+
+1. Copy the `.sst` file that you created to a domain controller.
+
+#### To distribute the list of trusted certificates by using Group Policy
+
+1. On the domain controller that has the customized `.sst` file, open the Group Policy Management Editor.
+
+1. In the Group Policy Management console, expand the **Forest**, **Domains**, and specific domain object that you want to modify. Right-select **Default Domain Policy GPO**, and then select **Edit**.
+
+1. In the navigation pane, under **Computer Configuration**, expand **Policies**, expand **Windows Settings**, expand **Security Settings**, and then expand **Public Key Policies**.
+
+1. Right-select **Trusted Root Certification Authorities**, and then select **Import**.
+
+1. In the Certificate Import Wizard, select **Next**.
+
+1. Enter the path and file name of the file that you copied to the domain controller, or use the **Browse** button to locate the file. Select **Next**.
+
+1. Confirm that you want to place these certificates in the **Trusted Root Certification Authorities** certificate store by selecting **Next**. select **Finish**. When you're notified that the certificates imported successfully, select **OK**.
+
+1. Close the Group Policy Management Editor.
+
+The policy is effective immediately, but the client computers must be restarted to receive the new settings, or you can type **gpupdate /force** from an elevated command prompt or from Windows PowerShell.
+
+## Registry settings modified
+
+The settings described in this document configure the following registry keys on the client computers. These settings aren't automatically removed if the GPO is unlinked or removed from the domain. These settings must be reconfigured, if you want to change them.
+
+| Registry keys | Value and Description |
+|--------|---------|
+| HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Microsoft\SystemCertificates\AuthRoot\DisableRootAutoUpdate | A value of 1 disables the Windows AutoUpdate of the trusted CTL. |  
+| HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Microsoft\SystemCertificates\AuthRoot\EnableDisallowedCertAutoUpdate | A value of 1 enables the Windows AutoUpdate of the untrusted CTL. |
+| HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\SystemCertificates\AuthRoot\AutoUpdate\RootDirUrl | Configures the shared location (the HTTP or the FILE path). |
+
+## Deleting Trusted and Untrusted CTLs
+
+It may be necessary for various reasons to delete all Trusted and Untrusted CTLs from a client machine. The following Certutil options can be used to accomplish this.
```

Comment:
`can be used to accomplish this.` - This isn't always clear to the reader when English is their second language or the text is localized. Could you use a noun or rephase?